source/Playnite.DesktopApp/ViewModels/SettingsViewModel.cs

```diff
@@ -233,6 +233,71 @@ public RelayCommand<object> WindowClosingCommand
             });
         }
 
+        public RelayCommand<object> AddSortingNameRemovedArticle
+        {
+            get => new RelayCommand<object>(_ =>
+            {
+                var res = dialogs.SelectString(
+                    resources.GetString("LOCEnterName"),
+                    resources.GetString("LOCAddNewItem"),
+                    "");
+                if (res.Result && !res.SelectedString.IsNullOrEmpty())
+                {
+                    if (settings.GameSortingNameRemovedArticles.Any(a => a.Equals(res.SelectedString, StringComparison.InvariantCultureIgnoreCase)))
+                    {
+                        dialogs.ShowErrorMessage(resources.GetString("LOCItemAlreadyExists"), "");
+                    }
+                    else
+                    {
+                        settings.GameSortingNameRemovedArticles.Add(res.SelectedString);
+                        settings.GameSortingNameRemovedArticles = settings.GameSortingNameRemovedArticles.GetClone();
+                    }
+                }
+            });
+        }
+
+        public RelayCommand<IList<object>> RemoveSortingNameRemovedArticle
+        {
+            get => new RelayCommand<IList<object>>((selectedItems) =>
+            {
+                foreach (string selectedItem in selectedItems)
+                {
+                    settings.GameSortingNameRemovedArticles.Remove(selectedItem);
+                }
+                settings.GameSortingNameRemovedArticles = settings.GameSortingNameRemovedArticles.GetClone();
+            }, (a) => a?.Count > 0);
+        }
+
+        public RelayCommand<object> FillSortingNameForAllGames
+        {
+            get => new RelayCommand<object>(_ =>
+            {
+                dialogs.ActivateGlobalProgress(args =>
+                {
+                    args.ProgressMaxValue = database.Games.Count;
+                    var updatedGames = new List<SDK.Models.Game>();
+                    var c = new SortableNameConverter(settings.GameSortingNameRemovedArticles, true);
+                    foreach (var game in database.Games)

```

Comment:
This should be done via `BufferedUpdate` (on `database`) to not call game update events for each game individually.